src/js/background.js

```diff
@@ -103,14 +103,21 @@
     return func || null;
   };
 
+  const registerToExtTST = async () => sendMsg(EXT_TST, {
```

Comment:
This looks not good - this should be defined somewhere under a namespace just for the external addon. Any ideas?

Reply:
You should move `sendMsg()` inside `handleExternalExtsRequirements` and on "ready" message call `handleExternalExtsRequirements` again?

Maybe in the future, even when handling other extensions requirements, a register message will be passed to TST, though.

Reply:
Sorry, I misunderstood as: `management.getAll()` don't return the information of TST until TST is loaded (even if it is installed). After some expreriments I've realized that it returns all installed extensions and `externalExts` always have TST's information even if TST is initialized after shareSNS. You're right, now I'm changing code to reuse `handleExternalExtsRequirements` itself.